napalm_eos/eos.py

```diff
@@ -795,9 +795,9 @@ def get_arp_table(self):
 
         return arp_table
 
-    def get_ntp_peers(self):
+    def get_ntp_stats(self):
 
-        ntp_peers = dict()
+        ntp_stats = list()
```

Comment:
Check this document, it was shocking for me when I found it:

http://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better